pkg/oc/cli/cmd/process.go

```diff
@@ -335,9 +324,7 @@ func RunProcess(f kcmdutil.Factory, in io.Reader, out, errout io.Writer, cmd *co
 	}
 	var version schema.GroupVersion
 	outputVersionString := kcmdutil.GetFlagString(cmd, "output-version")
-	if len(outputVersionString) == 0 {
```

Comment:
Can you leave here some TODO, so we don't forget about it. If this passes tests, I can live with it, we'll fix it while wiring the new printer.